### ISSUE TITLE
[CartBundle] - Feature Request - Cart Item - Redirect After Error

### ISSUE THREAD

**Issue by bretto36** [Sylius/Sylius#5683]
Is it possible to add another function for redirectAfterError on CartItemController. In my site i have a form that has additional fields, if they don't fill in the values i needed them to be redirected back to the product page, but on success still go to the cart. Could be useful as a default. my code below

`private function redirectAfterError(RequestConfiguration $configuration)
    {
        $request = $configuration->getRequest();

        if ($request->query->has('_error_redirect_to')) {
            return $this->redirectHandler->redirect($configuration, $request->query->get('_error_redirect_to'));
        }

        return $this->redirectToCartSummary($configuration);
    }`

All i did was change the redirectAfterAdd in the addAction if an error had occurred

**Comment by gabiudrescu** [comment]
@bretto36 is it something similar to what I have described here #5687 ?

**Comment by bretto36** [comment]
Yes similar issue. I've added the code so you can override the error redirect. But not so it automatically comes back to the same page. I can add something to the form on my branch to fix your issue too

**Comment by bretto36** [comment]
Check out the pull request mentioned above. Try and merge that in to your code and see if that solves your issue

**Comment by gabiudrescu** [comment]
@bretto36 it's not a bug that affects our current website, but I wanted to document it properly so the community should be aware of it.

so if you have solved it already, I'll close my issue - for the sake of a clean Sylius backlog.

**Comment by bretto36** [comment]
@gabiudrescu Thanks for trying to keep it all clean and tidy. Plenty of issues that need to be resolved. Glad i could help by closing one
